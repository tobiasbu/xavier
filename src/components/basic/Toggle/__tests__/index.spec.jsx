import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Toogle from '..';

describe('Toogle', () => {
  it('should render Toogle', () => {
    const component = shallow(
    <Toogle checked label="VocÃª quer ir ao cinema?" title="Marcar um cinema com os amigos?" />
    );
    expect(toJSON(component)).toMatchSnapshot();
  });
});
