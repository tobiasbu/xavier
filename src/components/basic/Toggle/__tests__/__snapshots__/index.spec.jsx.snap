// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toogle should render Toogle 1`] = `
<div
  className="a-toggle mars-neptune "
>
  <label
    htmlFor="m-i-a6ba504"
    id="m-l-a6ba504"
  >
    VocÃª quer ir ao cinema?
  </label>
  <input
    aria-checked={true}
    aria-labelledby="m-l-a6ba504"
    checked={true}
    id="m-i-a6ba504"
    name="voce-quer-ir-ao-cinema"
    onChange={[Function]}
    title="Marcar um cinema com os amigos?"
    type="checkbox"
  />
  <span
    className="a-toggle__shape"
  />
</div>
`;
